Update JS lesson materials (#30)
rrcobb/wf-2: src/lessons/javascript-fundamentals/loops.md

# Loops

> When you learnt the Python programming language, you saw how to use loops, such as the `for` construct to perform repititive tasks. In JavaScript, the provision for loops also exist, however there are some differences.

Author: ```suggestion
Loops are a way to perform repetitive tasks in JavaScript.
```

Watch this video to have an overview of loops in JavaScript

## The `for` loop.

<aside>

<strong>

Watch this video on the `for` loop 
</strong>
</aside>
<iframe width="750" height="315" src="https://www.youtube.com/embed/jS4aFq5-91M?start=10631&end=10835" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

## The `while` and `do while` loops.
<aside>

<strong>

Watch this video on the `while` and `do while` loops
</strong>
</aside>
<iframe width="750" height="315" src="https://www.youtube.com/embed/gTdesbu8nyo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

> ##More content here

Author: ```suggestion
```
